tests/scene/checkChildrenDidChange.tests.ts

```diff
@@ -9,13 +9,13 @@ describe('checkChildrenDidChange', () =>
         const container = new Container();
         const changeId = getChangeId(container);
 
-        expect(changeId).toEqual(16777216);
+        expect(changeId).toEqual(0);
 
         container._didContainerChangeTick = 4096;
```

Comment:
The test will fail if you change it to 4095. The weird math (`(x % 0xFFF) & 0xFFF`) just makes it wrap 1 earlier.

> (4095 % 0xFFF) & 0xFFF === 0

`(x % 0xFFF) & 0xFFF` isn't any better than `x & 0xFFF`. In fact it's a tiny bit worse.

Reply:
oh yeah! Silly me, I'Il change it back to your suggestion!

I think ultimatly the chances of this happening are pretty slim as you would need to do _exactly_ 4096 transforms between calling getLocalBounds().

Went for this approach so that checking for local bounds changes can be as fast as possible for effective caching. 

What do ya think, acceptible risk?

Reply:
It doesn't feel super low. We should at least document it.

Do we need the upper 8 bits for the `uid`: `((container.uid & 255) << 24)`? If we do need it, I feel like there's another case where we have chance the the bounds could be wrong. After all there's a 1 in 256 chance that the upper 8 bits collide. If we don't need it we could use the 16 instead of 12 bits for each id, which would reduce the chances significantly that we have a collision.